pkgs/development/python-modules/pysam/default.nix

```diff
@@ -75,6 +76,7 @@ buildPythonPackage rec {
 
     # Deselect tests that are known to fail due to upstream issues
     # See https://github.com/pysam-developers/pysam/issues/961
+    # See https://github.com/pysam-developers/pysam/issues/1151
     py.test \
```

Comment:
We should use `pytestCheckHook` and `pytestFlagsArray`/`disabledTests` instead.

Reply:
I have changed the commit message and read the message at https://github.com/NixOS/nixpkgs/blob/09b8e7b19320aef0b62312dc5f8668d4ef660ba6/pkgs/development/python-modules/pysam/default.nix#L56-L57 which is why I followed the rest of the disabled tests in the file